test/metabase/test_runner/junit/write.clj

```diff
@@ -130,7 +131,9 @@
      {:classname  (name (ns-name (:ns result)))
       :name       (name (symbol (:var result)))
       :time       (/ (:duration-ms result) 1000.0)
-      :assertions (:assertion-count result)}
+      :assertions (:assertion-count result)
+      ;; NOCOMMIT
```

Comment:
did you mean for a nocommit to end up committed?

Reply:
yeah, I was testing whether CircleCI would show the `system-out` attribute in JUnit output. If it did, I was going to write some fancy code to capture System out and System err and include it in JUnit output. Doesn't look like it does tho. So I'll remove this in a second